python/grass/jupyter/init_funcs.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
Let's rename this file to `setup`. `funcs` that's kind of clear as usually it is functions or functionality. `init.py` as file name would be little confusing in Python due to `__init__.py` file. Two words are fine especially for internal modules, but I don't have a better idea now, `setup` is okay, already used for the `init` function you are wrapping, so let's go with that.

Reply:
When I change the name to setup.funcs.py, I get an error in the Jupyter Notebook:

`ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-3-9dda28f7b25c> in <module>
      1 import grass.script as gs
----> 2 import grass.jupyter as gj

~/dist.x86_64-pc-linux-gnu/etc/python/grass/jupyter/__init__.py in <module>
      1 from __future__ import absolute_import
      2 
----> 3 from .setup.funcs import *

ModuleNotFoundError: No module named 'grass.jupyter.setup'`

Is there a better/correct way to call setup.funcs that doesn't result in this error? Or should be named something different?

Reply:
Try to parse my comment again and stop the first sentence right with the first period-space :-)

Reply:
Oh my gosh - I see it now. Thank you.